core/models/resource_request.py

```diff
@@ -1,18 +1,26 @@
 """
 resource_requests - requests to change a users resource
 """
+from uuid import uuid4
 from django.db import models
+from django.utils import timezone
+from core.models.user import AtmosphereUser
+from core.models.status_type import StatusType
 
-from core.models.abstract import BaseRequest
 
-
-class ResourceRequest(BaseRequest):
+class ResourceRequest(models.Model):
```

Comment:
I'm for removing `BaseRequest` altogether in a separate PR...

Reply:
I think BaseRequest could make sense if membership gets removed. I think this model could become the new BaseRequest, with a refactor.